src/Core/src/Platform/Android/Resources/values/colors.xml

```diff
@@ -3,5 +3,6 @@
     <color name="colorPrimary">#2c3e50</color>
     <color name="colorPrimaryDark">#1B3147</color>
     <color name="colorAccent">#3498db</color>
+    <color name="colorActionMenuTextColor">#FFFFFF</color>
```

Comment:
Currently in non Forms Shell apps this value is only settable through style attributes. The default in Forms is white so we are using white here